### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Birko15** [MrMeowzz/Funkin-MrMeowzz#48] [Q&A]
Hi
i would like to know how did you make your own html version in github
Respond if you can
Thanks.

**Comment by MrMeowzz** [comment]
Like what exactly about making a html version? Being able to play it or something else?

**Comment by Birko15** [comment]
Compiling the game and being able to play it

**Comment by MrMeowzz** [comment]
Well I just used github pages to be able to play it and to compile it you just run 'lime test html5 -release'

**Comment by Birko15** [comment]
k 
I´ll try that

**Comment by Birko15** [comment]
ok
i compiled the files 
do i just manually put all of the files in github or is there do i have to do somethings else.

**Comment by MrMeowzz** [comment]
Yes

**Comment by Birko15** [comment]
k

**Comment by Birko15** [comment]
Whenever i load the game the screen goes black is there something else that i´m doing wrong?

**Comment by MrMeowzz** [comment]
Do you have all of the html files uploaded?

**Comment by Birko15** [comment]
yes

**Comment by Birko15** [comment]
nevermind you were right i missed the txt file

**Comment by MrMeowzz** [comment]
oh

**Comment by Birko15** [comment]
but the resolution does not fit the whole screen
is there a fix

**Comment by MrMeowzz** [comment]
is resizeable set to true in the project.xml file for html5

**Comment by Birko15** [comment]
no
